package.json

```diff
@@ -11,7 +11,7 @@
     "build": "npm run extract-roles && babel -d dist src --copy-files",
     "prepare": "npm run build",
     "test": "ava",
-    "lint": "eslint src test && documentation lint src test",
+    "lint": "eslint src test/commands && documentation lint src test/commands",
```

Comment:
Just noticed this, but it seems unnecessary to run `documentation lint` on the tests 😄

Reply:
I'll include this change in https://github.com/aragon/aragon-cli/pull/359